subiquity/server/controllers/filesystem.py

```diff
@@ -881,6 +922,9 @@ def convert_autoinstall_config(self, context=None):
                             "'layout' and 'config', using 'layout'")
             self.run_autoinstall_guided(self.ai_data['layout'])
         elif 'config' in self.ai_data:
+            if self._system is not None:
```

Comment:
the 'mode' key is also invalid, a similar check would be nice

Reply:
ok